lib/themes/dosomething/paraneue_dosomething/js/components/Button.js

```diff
@@ -0,0 +1,37 @@
+const React = require('react');
+
+/**
+ * Button Component
+ * <Button />
+ */
+const Button = (props) => {
+    const classes = ['button'].concat(props.classes);
```

Comment:
well, I pass in the `-reactive` class because I need to differentiate between a button w/ the checkmark stuff added to it, and one without.